tests/apichecks/crds_test.go

```diff
@@ -233,6 +234,56 @@ func TestCRDsAcronyms(t *testing.T) {
 	test.CompareGoldenFile(t, "testdata/exceptions/acronyms.txt", want)
 }
 
+// Avoid passing sensitive data as plain text in the CRD
+func TestNoSensitiveField(t *testing.T) {
+	crds, err := crdloader.LoadAllCRDs()
+	if err != nil {
+		t.Fatalf("error loading crds: %v", err)
+	}
+
+	var errs []string
+
+	sensitiveKeywords := []string{
```

Comment:
Might be worth pulling this up to a constant list at the top rather than embedding inside a function at line 246. This seems like something we are going to want to easily grow as we find other field names which suggest a problem. (passphrase, key, private, pemCertificate, ....)